htmlmaker_postprocessing.js

```diff
@@ -95,6 +106,83 @@ fs.readFile(file, function editContent (err, contents) {
     $(this).after(match[4]);
   });
 
+  // remove Section-Blank-Page sections
+  var sectionBlankPage = $("section.blankpage");
+  sectionBlankPage.remove();
+
+  //// Strip pageBreaks preceding Section starts: Part 1 (of 2):
+  // catch & remove any page break directly preceding Section Starts
+  //  (nothing should be outside of a seciton block, but just in case)
+  var leadingPageBreak = $("section").prev(".PageBreakpb");
+  // cycle through each Page Break para directly preceding a Section Start
+  leadingPageBreak.each(function() {
+    // capture any Page Breaks directly preceding the selected one
+    var consecutiveLeadingPBs = $(this).prevUntil(":not(.PageBreakpb)").addBack();
+    consecutiveLeadingPBs.remove();
+  });
+
+  //// Strip pageBreaks preceding Section starts: Part 2 (of 2):
```

Comment:
I'm torn re: needing to ensure the sections are followed by another section or div, since there really shouldn't be any floating text on it's own without a section or div parent. Better safe than sorry I guess. A simpler way to do this is: $("section:has(.PageBreakpb:last-child) + section, section:has(.PageBreakpb:last-child) + div, div:has(.PageBreakpb:last-child) + section, div:has(.PageBreakpb:last-child) + div").prev().children().last().remove();

Also, we'll again assume for now that stacked pagebreaks are intentional.